src/aspire/abinitio/commonline_sync.py

```diff
@@ -179,7 +189,7 @@ def _syncmatrix_ij_vote(self, clmatrix, i, j, k_list, n_theta):
 
         rots = self._rotratio_eulerangle_vec(clmatrix, i, j, good_k, n_theta)
 
-        if rots.shape[0] > 0:
```

Comment:
Could you comment briefly on this change and the one below (regarding `c_alpha`)?

Reply:
Sure. When attempting to get a baseline output from `orient3d` in the command line before making changes, I found that `orient3d` was throwing an error. Here is the command I ran:

`$ aspire orient3d --starfile_in tests/saved_test_data/sample_relion_data.star --starfile_out junk.star` 

Here is the error: `ValueError: Invalid shape in axis 0: 0.`.

After some inspection, it turns out that `_get_cos_phis` was passing `c_alpha` with length zero, causing an invalid `angles` shape of `(0, 3)` to be passed to scipy's `Rotation`. The added code throws out these values.